examples/hubert/loss/hubert_loss.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+import torch
+import torch.nn.functional as F
+from torch import Tensor
+
+
+def hubert_loss(
+    logit_m: Optional[Tensor],
+    logit_u: Optional[Tensor],
+    feature_pen: Tensor,
+    masked_weight: float = 1.0,
+    nomask_weight: float = 0.0,
+    feature_weight: float = 10.0,
+    reduction: str = "sum",
+) -> Tensor:
+    """Compute the cross-entropy loss on HuBERT masked and non-masked logits.
+    Args:
+        logit_m (Tensor or None): The masked logit Tensor of dimension `[masked_frames, final_dim]`.
```

Comment:
should we standardize using tuples to represent shapes? e.g. `(masked_frames, final_dim)`

Reply:
sure, will change it to `(masked_frames, final_dim)`